commit b4c6b246594ccc8b190215f43243fa8926441e64
Author: K. Y. Srinivasan <kys@microsoft.com>
Date:   Wed Dec 7 01:16:28 2016 -0800

    Drivers: hv: vmbus: Prevent sending data on a rescinded channel
    
    commit e7e97dd8b77ee7366f2f8c70a033bf5fa05ec2e0 upstream.
    
    After the channel is rescinded, the host does not read from the rescinded channel.
    Fail writes to a channel that has already been rescinded. If we permit writes on a
    rescinded channel, since the host will not respond we will have situations where
    we will be unable to unload vmbus drivers that cannot have any outstanding requests
    to the host at the point they are unoaded.
    
    Signed-off-by: K. Y. Srinivasan <kys@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1319_b4c6b246594ccc8b190215f43243fa8926441e64.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1319_b4c6b246594ccc8b190215f43243fa8926441e64.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
